pkg/reconciler/shared/tektonconfig/tektonconfig.go

```diff
@@ -224,8 +223,8 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, tc *v1alpha1.TektonConfi
 
 	tc.Status.MarkPostInstallComplete()
 
-	// Update the object for any spec changes
-	if _, err := r.operatorClientSet.OperatorV1alpha1().TektonConfigs().Update(ctx, tc, metav1.UpdateOptions{}); err != nil {
+	// Update the object status
```

Comment:
```suggestion
	// Update the object status for any spec changes 
```

Reply:
Update the object status for any spec changes  is a bit confuising
operator should not change the spec
if a user changes the spec, we reoncile, we dont just update the status